README.md

```diff
@@ -1,3 +1,7 @@
+<img src="https://i.ibb.co/nPLDJ12/Screen-Shot-2020-10-22-at-18-54-45.png"></img>
+
+
+
```

Comment:
Why the extra spacing here? There should never be more than one line break consecutively, per our general formatting guidelines. When we add a CONTRIBUTING section I'll make this clearer :)